docs/user-guide/fhir-server-config.md

```diff
@@ -0,0 +1,23 @@
+
+# IBM FHIR Server configuration steps
+
+Collected here are notes on how to configure an IBM FHIR server to support the Alvearie cohorting common component. Many of the configuration steps, such as how to configure persistence, are left up to the solution consuming the cohorting capability, but this document describes some specific configuration that should be done if cohorting is going to be used.
+
+## Create a tenant for storing knowledge artifacts
+The measure evaluation engine relies on Measure and Library definitions that are assumed to be stored in an R4 compliant FHIR server. When using a multitenant IBM FHIR server installation, it is recommended that solutions consuming the engine configure a single, shared tenant in their FHIR server that will contain all knowledge artifacts. In our development environment, we call this tenant "knowledge". See the [IBM FHIR Server User's Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide/#322-tenant-specific-configuration-properties) for complete details on how to configure an additional tenant.
+
+## Enable searching for ValueSet resources vs. those defined in an IG
+In the fhir-server-config.json for the knowledge tenant created above, set fhirServer/core/serverRegistryResourceProviderEnabled to true to enable user-defined ValueSet resources to be searchable via the REST API. This is a required setting to enable search functionality in the cohort and measure authoring tool. See [the Dev Guide](/dev-guide/value-sets?id=fhir-server-setup) for complete details. The property is also described in the [IBM FHIR Server User's Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide/).
```

Comment:
the link for "the Dev Guide" is not found. Gets a 404 when clicking on it.

Reply:
That isn't going to work unless it is processed by docsify first. You can test by checking out the branch and running ``docify serve docs`` from the root of the project if you are interested. I just did it to make sure and it was fine.